src/main/resources/db/migration/V12__index-arena-data.sql

```diff
@@ -0,0 +1 @@
+CREATE UNIQUE INDEX arena_data_table_name_arena_id_idx on arena_data (arena_id, arena_table_name);
```

Comment:
Er dette en unik kombinasjon egentlig? Lurer på om vi bør fjerne unique. Hensikten er vel uansett å få på index så spørringer ikke går så tregt? :)

Reply:
Hmm.. jeg hadde egentlig trodd at det skulle funke med unique men det er kanskje ikke nødvendig

Reply:
Det er noe med testene som gjør at det ikke funker. Fant ikke helt ut av det utenom at hvis man kjører DeltakerIntegrationTest og deretter RetryArenaMessageProcessorService så feiler de siste testene. Får dog ikke en sql feil som på github når jeg kjører lokalt, lokalt blir feilen at en (av 3) deltakere ikke går fra status RETRY-> HANDLED. Merger denne uten unique da det fungerer og tror ikke det har så stor verdi å bruke for lang tid på det.